pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -297,7 +365,9 @@ stdenv.mkDerivation rec {
     cp ./bazel_src/scripts/zsh_completion/_bazel $out/share/zsh/site-functions/
   '';
 
-  doInstallCheck = true;
+  # Temporarily disabling for now. A new approach is needed for this derivation as Bazel
+  # accesses the internet during the tests which fails in a sandbox.
+  doInstallCheck = false;
```

Comment:
Is this needed if we have `bazel.tests` anyway? Perhaps we can skip this step permanently?

Reply:
I think once we have a test for Java & C++ rules, we can get rid of these tests alltogether.